README.md

```diff
@@ -17,7 +17,7 @@
 
 ## ✨ Highlight features
 
-1. **Visual Builder** to get started quickly and iterate. 
+1. **Visual Builder** to get started quickly and iterate.
 1. **Access to Code** so developers can tweak any component using Python.
 1. **Playground** to immediately test and iterate on their flows with step-by-step control.
 1. **Multi-agent** orchestration and conversation management and retrieval.
```

Comment:
At minimum, these should be bullets, but I think the wording could be stronger as well.

```suggestion
- **Visual Builder** to get started quickly and iterate.
- **Access to Code** so developers can tweak any component using Python.
- **Playground** to immediately test and iterate on their flows with step-by-step control.
- **Multi-agent** orchestration and conversation management and retrieval.
```

```suggestion
- **Visual builder interface** to quickly get started and iterate .
- **Source code access** lets you customize any component using Python.
- **Interactive playground** to immediately test and refine your flows with step-by-step control.
- **Multi-agent orchestration** with conversation management and retrieval.
```